### PR TITLE
Fix some broken links, and streamline docs

### PR DESCRIPTION
**Pull request by glennj**
- docs/TESTS.md: same commands were repeated three times.
- docs/RESOURCES.md: populate from exercises/shared/.docs/help.md
- docs/INSTALLATION.md: broken link

### CODE REVIEW

File: docs/RESOURCES.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+    - [Join Discord][join-discord]
+- [StackOverflow][stackoverflow]
+- Exercism's resources
+    - get help on [the forum][exercism-forum], or
+    - in the #support channel on [Exercism's discord][exercism-discord].
+
+
+[getting-started]: https://ballerina.io/learn/get-started-with-ballerina/
+[ballerina-examples]: https://ballerina.io/learn/by-example/
+[join-discord]: https://ballerina.io/community/#ballerina-discord-community
```
Comment (ThisaruGuruge) [inline docs/RESOURCES.md:17]:
```suggestion
[join-discord]: https://discord.gg/ballerinalang
```

File: docs/TESTS.md [hunk 2]
```diff
@@ -1,132 +1,39 @@
...
 
+	The directory location depends on your OS and your configured exercism workspace.
+	See `exercism configure` for your workspace location.
 
 3. Initialize the project prior to executing tests:
 
-     ```batchfile
-     /Users/johndoe/exercism/ballerina $ ballerina init
+     ```sh
+     ballerina init
```
Comment (ThisaruGuruge) [inline docs/TESTS.md:21]:
```suggestion
     bal init
```

File: docs/TESTS.md [hunk 3]
```diff
@@ -1,132 +1,39 @@
...
      ```
-
+     
 4. Run the tests:
 
-    ```
-    /Users/johndoe/exercism/ballerina $ ballerina test hello-world
-    ```
+     ```sh
+     ballerina test --offline
```
Comment (ThisaruGuruge) [inline docs/TESTS.md:27]:
```suggestion
     bal test --offline
```

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
@ThisaruGuruge Any objections to this being merged?

**Review by ThisaruGuruge — APPROVED** [review]

**Comment by ThisaruGuruge** [comment]
> @ThisaruGuruge Any objections to this being merged?

All good. Merged.

**Pull request merged**